### PR TITLE
1) ViT to .eval() mode 2) checkpoint save for each 10th epoch under ./checkpoints/

### PR DESCRIPTION
**Pull request by justin4ai**
Please review mine

### CODE REVIEW

**Comment by justin4ai** [comment]
Also I think we need to periodically (probably for each epoch) print accuracy scores on validation set (reasonably 10% of full training set) while training.

File: train.py [hunk 1]
```diff
@@ -8,25 +8,25 @@
...
     classifier.to(device)
-    num_epochs = 100
+    num_epochs = 50
     optimizer = torch.optim.Adam(classifier.parameters(), lr = 0.01)
     checkpoint = None
     
-    if glob.glob("./*.pt"):
-        checkpoint = torch.load("./model_state_dict.pt")
+    if glob.glob("./*.pt"): # never be run as long as we save all the checkpoints under 'checkpoints' folder
+        checkpoint = torch.load("./detector.pt")
```
Comment (JihongJeong) [inline train.py]: if glob.glob("./checkpoints/*.pt") # this came out while train session interrupted  
    checkpoint_path = glob.glob.pop() # bring latest checkpoint

Comment (justin4ai) [reply train.py]: I think your review broke a bit? I cannot recognize what you want. Would you mind re-writing it?

File: train.py [hunk 2]
```diff
@@ -8,25 +8,25 @@
...
     print(device, torch.cuda.get_device_name())
     classifier.to(device)
-    num_epochs = 100
+    num_epochs = 50
     optimizer = torch.optim.Adam(classifier.parameters(), lr = 0.01)
     checkpoint = None
     
-    if glob.glob("./*.pt"):
-        checkpoint = torch.load("./model_state_dict.pt")
+    if glob.glob("./*.pt"): # never be run as long as we save all the checkpoints under 'checkpoints' folder
```
Comment (JihongJeong) [inline train.py:22]:
```suggestion
    if glob.glob("./checkpoints/*.pt"): 
```
I used this session to restart training when it was interrupted before the training is done for all epochs. So not running this before all checkpoints are saved doesn't make sense.

File: train.py (hunk 1 above)
Comment (JihongJeong) [inline train.py]:
```suggestion
        checkpoint_path = glob.glob.pop() # bring latest checkpoint
        checkpoint = torch.load(checkpoint_path)
```
Although we should know the latest checkpoint to restart the training when it get interrupted, current checkpoint will be the last component of the checkpoint list.

**Review by JihongJeong — APPROVED** [review]
